packages/editor-ui/src/components/ResourceMapper/ResourceMapper.vue

```diff
@@ -568,5 +583,50 @@ defineExpose({
 				})
 			}}
 		</N8nNotice>
+		<div
+			v-if="showTypeConversionOptions && state.paramValue.schema.length > 0"
+			:class="$style.typeConversionOptions"
+		>
+			<ParameterInputFull
+				:parameter="{
+					name: 'attemptToConvertTypes',
+					type: 'boolean',
+					displayName: 'Attempt to convert types',
+					default: false,
+					description: 'Attempt to convert types when mapping fields',
+				}"
+				:path="props.path + '.attemptToConvertTypes'"
+				:value="state.paramValue.attemptToConvertTypes"
+				@update="
+					(x: IUpdateInformation<NodeParameterValueType>) => {
+						state.paramValue.attemptToConvertTypes = x.value as boolean;
+						emitValueChanged();
+					}
+				"
+			/>
+			<ParameterInputFull
+				:parameter="{
+					name: 'ignoreTypeMismatchErrors',
+					type: 'boolean',
+					displayName: 'Ignore Type Mismatch Errors',
+					default: false,
+					description: 'Whether type mismatches should be ignored, rather than returning an Error',
+				}"
+				:path="props.path + '.ignoreTypeMismatchErrors'"
+				:value="state.paramValue.ignoreTypeMismatchErrors"
+				@update="
+					(x: IUpdateInformation<NodeParameterValueType>) => {
+						state.paramValue.ignoreTypeMismatchErrors = x.value as boolean;
+						emitValueChanged();
+					}
+				"
+			/>
+		</div>
 	</div>
 </template>
+
+<style module lang="scss">
+.typeConversionOptions {
```

Comment:
I suggest we increase the spacing a bit both around these options and inside it:
```suggestion
.typeConversionOptions {
	display: grid;
	padding: var(--spacing-m);
	gap: var(--spacing-2xs);
}
```

Reply:
This is so much better 🙏